dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -314,13 +312,12 @@ else
                              select(.name == \"CHE_JDBC_USERNAME\").valueFrom) |= {\"configMapKeyRef\":{\"key\":\"che.jdbc.username\",\"name\":\"rhche\"}}")
 fi
 
-#echo "$CHE_APP_CONFIG_YAML" | oc process -f - | cat
 if ! (echo "$CHE_APP_CONFIG_YAML" | oc process -f - | oc apply -f - > /dev/null 2>&1); then
   echo -e "\033[91;1mFailed to process che config\033[0m"
   exit 1
 fi
 
-CHE_STARTUP_TIMEOUT=120
+CHE_STARTUP_TIMEOUT=256
```

Comment:
why 256 seconds ?

Reply:
Had to increase because it was timing out. And I don't know, 256 is a power of two :D

Reply:
Can do `300`

Reply:
changed to 300